content/03.use-cases-a.md

```diff
@@ -5,12 +5,12 @@
 Many researchers often start their use of GitHub to backup their working code and data to a remote server (Just push and pull, see [Box 1](#definitions), from their own repo). 
 The GitHub repository serves as an additional backup and also allows the user to synchronize files and work on any computer with internet access. 
 Because GitHub is built on git, version control features are also available, allowing users to re-visit or even restore previous versions of the repository.
+
 Although it is not recommended to use GitHub for long term data archiving, GitHub integrates with Zenodo, a CERN-funded long-term data archiving service. Zenodo can be linked to your GitHub account and archival activated for any public repository. Then, every time a release ([Box 1](#definitions)) is made, a snapshot of the entire repository is archived in Zenodo with a versioned, citable DOI that can be mentioned in a Data Availability Statement (see 'Making code citable' below for further discussion).
 Unfortunately, both GitHub and Zenodo have file size limits. 
-For GitHub, commits (see [Box 1](#definitions)]) greater than 50 MB receive a warning and there is a hard block on commits larger than 100 MB (@https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github). 
-Large datasets therefore may not be able to pushed up to GitHub repositories. 
-Data greater than 50 MB can be stored in any one of many data repositories that can be integrated or linked with GitHub (e.g., osf.io) or one can make use of large file storage from GitHub (@https://git-lfs.github.com; which may have some costs associated with it). 
-Zenodo has a similar limit of 50GB per dataset, but the user can upload multiple datasets and in addition the Zenodo help files suggest that larger datasets can be accommodated if users contact Zenodo (again a cost may be associated with this; see @https://help.zenodo.org).
+GitHub is not recommended for storing large files and thus, commits (see [Box 1](#definitions)]) greater than 50 MB receive a warning and commits larger than 100 MB are blocked (@https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github). 
```

Comment:
Great, thanks for making the edit here. I fixed a spacing issue I noticed and typo